README.md

```diff
@@ -8,36 +8,55 @@ description: >-
 # Catalyst Product Management Curriculum
 
 Developed by Founders and Coders in collaboration with CAST and Catalyst.
-
-The Product Management Curriculum was validated as part of Collaborative Digital Training (May - June 2020).
+The Product Management Curriculum was validated as part of **Collaborative Digital Training** (May - June 2020) and the **Discovery Learning Programme** (Sept - Oct 2020).
 
 ## Learning Outcomes
 
 - Understand the product development cycle from discovery and definition to build phase
 - Understand the role and responsibilities of a Product Manager and a product owner
-- Understand Scrum and Agile
 - Learn methods and techniques that facilitate user-centred, consensus-driven decision making in Definition, Discovery and Build phase
```

Comment:
```suggestion
- Learn methods and techniques that facilitate a user-centred approach to Discovery, Definition, Prototyping and Building. 
```

I know you didn't write this (it was me lol) but could we say this instead?

Reply:
of course!